### PR TITLE
Articles added and index updated

### PR DESCRIPTION
**Pull request by basalumutgazi**
12 articles are added.
The index file has been updated:
2 new groups and 12 new articles are added.

### CODE REVIEW

File: doc/source/60s.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# 6️⃣0️⃣ THE 60s
```
Comment (jbampton) [inline doc/source/60s.md:1]:
```suggestion
# ✊🏿6️⃣0️⃣ THE 60s
```

**Comment by BaseMax** [comment]
You are back stronger than before. welcome mate.

**Review by BaseMax — APPROVED** [review]
LGTM.

**Comment by jbampton** [comment]
Thanks @BaseMax  

Restyled PR is now merged

**Comment by basalumutgazi** [comment]
> You are back stronger than before. welcome mate.

Thank you mate!

**Pull request closed without merging**